package.json

```diff
@@ -76,16 +76,16 @@
     "vkbeautify": "^0.99.3"
   },
   "peerDependencies": {
-    "pixi.js": "^5.0.3",
+    "pixi.js": "^6.3.0",
```

Comment:
This is kind of surprising to me that this is the only pixi change. I expected also to see pixi change in our dev dependencies, but we already have pixi@6 listed there... Maybe we have already been using pixi@6 for development and testing for while?